rmgpy/molecule/translator.py

```diff
@@ -531,6 +531,8 @@ def _write(mol, identifier_type, backend):
             logging.debug('Backend {0} is not able to generate {1} for this molecule:\n'
                           '{2}'.format(option, identifier_type, mol.to_adjacency_list()))
 
+    logging.error('Unable to generate identifier for this molecule:\n{0}'.format(mol.to_adjacency_list()))
```

Comment:
Maybe instead add the adjlist to the error message below?

Reply:
I thought about doing that originally. One consideration is that with the current setup, logging goes into RMG.log, while the error goes into stderr, which is only printed to the terminal (or slurm/sge output files). I can move it though.

Reply:
Fair enough, then let's keep it. I'm OK with all other commits too.

Reply:
Might not be relevant here but the discussion reminded me:  I am a fan of using `logging.exception()` in an exception handler to add context and  make sure the original stack trace ends up in the log.  Can then `raise()` the original exception as if you hadn’t caught it

Reply:
Might not be relevant here but the discussion reminded me:  I am a fan of using `logging.exception()` in an exception handler to add context and  make sure the original stack trace ends up in the log.  Can then `raise()` the original exception as if you hadn’t caught it

Reply:
Yes, I was actually hoping that this would be a good opportunity to use logging.exception(), but that has to be called from an exception handler. It seemed easier to log this message once in this function rather than logging the exception in each function which calls it.

I do think there are a lot of instances where we use `logging.error` when catching exceptions where `logging.exception` could be used instead.